scipy/_lib/tests/test_warnings.py

```diff
@@ -40,7 +40,21 @@ def visit_Call(self, node):
         ast.NodeVisitor.generic_visit(self, node)
 
         if p.ls[-1] == 'simplefilter' or p.ls[-1] == 'filterwarnings':
-            if node.args[0].value == "ignore":
+            # get first argument of the `args` node of the filter call
+            match node.args[0]:
+                case ast.Constant() as c:
```

Comment:
Since we dropped 3.9 in https://github.com/scipy/scipy/pull/20373, I'm assuming that `match:` statements are now fair game.